addon/components/file-browser/style.scss

```diff
@@ -62,7 +62,7 @@
 
         background: #efefef;
         border: 1px solid #eee;
-        margin: 0 0 10px;
+        margin: 0 0 1px;
```

Comment:
Please change back to 10px margin-bottom and add a class (e.g. `small-view`) that overrides to 1px.